### PR TITLE
feat(docs): time-series blog post

### PR DESCRIPTION
**Pull request by amyshwang**
1. Deleting the old content
2. Adding new file with the URL time-series-data based on the draft
3. Updating content
4. Adding links to demo and descriptions to additional reading links

### CODE REVIEW

File: blog/2023-01-18-time-series-data.mdx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+<Screenshot
+  alt="Architecture of the storage model with column files and time partitions"
+  height={323}
+  src="/img/blog/2023-01-18/model.png"
+  title="Data stored in columns and partitioned by time"
+  width={650}
+/>
+
+We didn't get everything right from the start! In 2021, we shipped
+[QuestDB 6.0](/blog/2021/05/10/questdb-release-6-0-tsbs-benchmark/) to support
```
Comment (bluestreak01) [inline blog/2023-01-18-time-series-data.mdx:286]: bad link description, e.g. "QuestDB 6.0" leads to article titled "How we achieved write speeds of 1.4 million rows per second". Perhaps paragraph needs to be rephrased to set user expectations for the link

File: blog/2023-01-18-time-series-data.mdx [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  ```questdb-sql title="Retrieving data from the table ‘trips’, where the column ‘pickup_datatime’ happens during the month of June 2018"
+  SELECT * FROM 'trips'
+  WHERE pickup_datetime IN '2018-06';
+  ```
+
+- **Time series joins**
+
+  Align join time-series data from two different tables, which do not have
+  exactly matching timestamps. These are known as
+  [`ASOF`](/docs/reference/sql/join#asof-join) joins. Below, the two tables,
```
[the remark this answers is not in the document]
Comment (bluestreak01) [reply blog/2023-01-18-time-series-data.mdx:187]: we need better link description here, perhaps "There are known as ASOF joins, we have [example of asof join](/docs/reference/sql/join/#asof-join) in our documentation

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: blog/2023-01-18-time-series-data.mdx [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+  SAMPLE BY 15m ALIGN TO CALENDAR;
+  ```
+
+- **Interval search**
+
+  Fast retrieving data over arbitrary intervals. For example, zooming into a
+  specific timeframe preceding a monitoring alert to better understand the
+  underlying cause in real-time. QuestDB’s
+  [`WHERE`](/docs/reference/sql/where#symbol-and-string) and
+  [`IN`](/docs/reference/sql/where/#time-range) for timestamp search is fast and
```
Comment (bluestreak01) [inline blog/2023-01-18-time-series-data.mdx:174]: Paragraph need to be re-phrased to ensure link is descriptive

File: blog/2023-01-18-time-series-data.mdx [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+  WHERE symbol = 'BTC-USD' AND timestamp > dateadd('d', -1, now())
+  SAMPLE BY 15m ALIGN TO CALENDAR;
+  ```
+
+- **Interval search**
+
+  Fast retrieving data over arbitrary intervals. For example, zooming into a
+  specific timeframe preceding a monitoring alert to better understand the
+  underlying cause in real-time. QuestDB’s
+  [`WHERE`](/docs/reference/sql/where#symbol-and-string) and
```
[the remark this answers is not in the document]
Comment (bluestreak01) [reply blog/2023-01-18-time-series-data.mdx:173]: same for `where` - link description is too short

File: blog/2023-01-18-time-series-data.mdx [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+  the disk rather than lifting the entire dataset. Partitioning also allows
+  having multiple tiers of storage, where older partitions can be mounted into
+  cold storage, which is cheaper and slower.
+
+- **Downsampling and interpolation**
+
+  Representing the data with a lower frequency. For example, shifting from a
+  daily view to a monthly view. In order to facilitate such queries with SQL and
+  make them less verbose, QuestDB built a native extension to ANSI SQL with the
+  keyword [`SAMPLE BY`](/docs/reference/sql/sample-by#examples). This SQL
```
[the remark this answers is not in the document]
Comment (bluestreak01) [reply blog/2023-01-18-time-series-data.mdx:152]: `SAMPLE BY` link description is too short

File: blog/2023-01-18-time-series-data.mdx [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+out from relational databases and other types. We provide a list of these
+must-have features for a TSDB. Then, we show how you can do this on QuestDB, an
+open-source time-series database with fast ingests and SQL queries. You can run
+the examples on the [QuestDB demo site](https://demo.questdb.io/) to check the
+query performance.
+
+- **Automated partitioning management**
+
+  Time partitions are created automatically as data arrives. In QuestDB, data is
+  [partitioned by time](/docs/concept/partitions/) (hourly, daily, weekly or
```
Comment (bluestreak01) [inline blog/2023-01-18-time-series-data.mdx:140]: link description is too short

File: blog/2023-01-18-time-series-data.mdx [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+author: Nicolas Hourcard
+author_title: Co-founder & CEO, QuestDB
+author_url: https://github.com/tris0laris
+author_image_url: https://avatars.githubusercontent.com/tris0laris
+description:
+  A primer on time-series data and time-series databases (TSDB), and why we
+  decided to build QuestDB
+image: /img/blog/2023-01-18/banner.jpg
+tags: [company, startup, entrepreneurship, story]
+slig: time-series-data
```
Comment (argshook) [inline blog/2023-01-18-time-series-data.mdx:14]:
```suggestion
slug: time-series-data
```

File: blog/2023-01-18-time-series-data.mdx [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+/>
+
+Time-series data has always been plentiful in financial services with
+fast-changing tick price data and in e-commerce/ad-tech to better understand
+user analytics. With the rise in connected devices, application monitoring, and
+observability, time-series data is now critical in nearly all fields. We list a
+couple of examples below:
+
+<Screenshot
+  alt="Example of time-series data use cases"
```
Comment (argshook) [inline blog/2023-01-18-time-series-data.mdx:72]:
```suggestion
  alt="Table showing time series data use cases by segment"
```

Same suggestion as from Vlad, just in a `suggestion` format for easy committing

File: netlify.toml [hunk 9]
```diff
@@ -278,6 +278,10 @@
   from = "/docs/third-party-tools/kafka/"
   to = "/docs/third-party-tools/kafka/overview/"
 
+[[redirects]]
+  from = "/blog/2020-11-26-why-timeseries-data/"
```
Comment (argshook) [inline netlify.toml:282]:
```suggestion
  from = "/blog/2020/11/26/why-timeseries-data/"
```

The date part is separated by forward slash. You can confirm this by checking other rules in this file that redirect blog post pages or other blog posts, like, for example [this one](https://deploy-preview-1307--unruffled-blackwell-31bfb2.netlify.app/blog/2023/01/03/change-data-capture-with-questdb-and-debezium) (notice how URL is separated with forward slash for date part)

**Review by argshook — CHANGES_REQUESTED** [review]

**Review by argshook — APPROVED** [review]

**Pull request merged**